.github/workflows/ci.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        php-version: [ '7.2', '7.3', '7.4', '8.0', '8.1' ]
-        symfony-version: ['^4.4', '^5.0']
+        php-version: ['8.1' ]
```

Comment:
__suggestion:__ If we set the requirement to `>=8.0` we should keep `8.0 `version in the CI :smile:

Reply:
Symfony 6.1 requires php 8.1, so if we keep php 8.0 in the CI, it fails. We may not use matrix to add 8.0 + symfo 6.0 in the CI.

Reply:
@sdechomets we can use `exclude` to not run `8.0` with `6.1` (see https://stackoverflow.com/a/68994907/1905446)

but actually, since we're making a big breaking change in this PR, I suggest to also drop support for `php8.0` so we should simply put `php >=8.1` and `symfony ~6.1` in composer, WDYT?

Reply:
I agree, as the need for compatibility has not been expressed yet, we can indeed start with a release including only the most recent versions of symfony and php, no need for a wider compatibility as far as I know.